addons/website/static/tests/tours/link_tools.js

```diff
@@ -0,0 +1,54 @@
+odoo.define("website.tour.link_tools", function (require) {
```

Comment:
I suggest to use the new odoo module syntax when creating a new js file.

Reply:
Thank you for the fix, it's cleaner. I tested and it works well :)